newIDE/app/src/Course/CoursePreviewBanner.js

```diff
@@ -57,7 +58,7 @@ const styles = {
     color: '#1D1D26',
   },
   gdevelopAvatar: { width: 20, height: 20 },
-  thumbnail: { borderRadius: 4, aspectRatio: '16 / 9' },
+  thumbnail: { borderRadius: 4, aspectRatio: '16 / 9', maxWidth: '100%' },
```

Comment:
this maxWidth feel hacky, would a object-fit: "contain" or similar make more sense?

Reply:
Not sure that's a hack. I don't know why, on mobile, without this, the image completely overflows the parent:
<img width="341" alt="image" src="https://github.com/user-attachments/assets/f232ba12-ae7b-4c4d-9737-1ac0ade10c53" />